MagicBall.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState} from "react"
+import styled from 'styled-components/native'
+
+const Quotes = [
+  {
+    answear: "It is certain.",
+  },
+  {
+    answear: "It is decidedly so.",
+  },
+  {
+    answear: "Without a doubt.", 
+  },
+  {
```

Comment:
since you only have one key-value pair in each object maybe it's enough to have the answers just as strings in the array?
such as:
```
const quotes = [
  'It is certain',
  'Without a doubt',
  '...'
]
```
Also, stick to either `const quotes` or `const QUOTES` it'd not a component so dont use Quotes with capital Q.